README.md

```diff
@@ -46,35 +46,9 @@ tenderseed --home /some/path/to/home/dir
 
 ## Configuration
 
-TenderSeed is configured by a [toml](https://github.com/toml-lang/toml) config file found in the tenderseed [home dir](#Home-Dir) as `config/config.toml`
+Use flags.  Statefulness bad.
```

Comment:
not really a fan of this. I would rather offer an example for the cli and the configuration. I can open a PR later today to do this.

Reply:
Bad idea predefine args in source code, it's not clear. May be use ENV variables prefer args for statefulness ?